README.md

```diff
@@ -1,111 +1,129 @@
-[![Documentation](https://readthedocs.org/projects/restic/badge/?version=latest)](https://restic.readthedocs.io/en/latest/?badge=latest)
+<p align="center">
+  <img src="https://restic.readthedocs.io/en/latest/_static/logo.png" alt="Restic logo">
+</p>
+
+<p align="center">
+  <strong>Restic</strong> – <strong>Fast</strong>, <strong>efficient</strong>, and <strong>secure</strong> backup program
+  <br>
+  <a href="https://restic.net/">Website</a> |
+  <a href="https://restic.readthedocs.io/">Documentation</a> |
+  <a href="https://forum.restic.net/">Forum</a> |
+  <a href="https://github.com/restic/restic/issues">Issues</a>
+</p>
+
+# Restic
+
+[![Documentation](https://readthedocs.org/projects/restic/badge/?version=latest)](https://restic.readthedocs.io/en/latest/)
 [![Build Status](https://github.com/restic/restic/workflows/test/badge.svg)](https://github.com/restic/restic/actions?query=workflow%3Atest)
 [![Go Report Card](https://goreportcard.com/badge/github.com/restic/restic)](https://goreportcard.com/report/github.com/restic/restic)
 
-# Introduction
+Restic is a program that does backups right and was designed with the following
+principles in mind:
```

Comment:
I prefer the first phrase of the original, because it gives a very concise summary: backup prog, fast, efficient, secure, Linux,maxOS, Windows, *BSD

Reply:
I've restored the original description.